llama-cli/src/main.rs

```diff
@@ -314,3 +317,81 @@ fn main() {
         }
     }
 }
+
+fn ad_hoc_test() {
```

Comment:
This function will be removed before we merge. It's just here so others can validate my results and test things.